tendrl-node-agent.spec

```diff
@@ -24,6 +24,7 @@ Requires: collectd
 Requires: python-jinja2
 Requires: tendrl-commons
 Requires: hwinfo 
+Requires: pytz
```

Comment:
I received the following error during service start
  File "/usr/lib/python2.7/site-packages/tendrl/commons/message.py", line 6, in <module>
    from tendrl.commons.utils.time_utils import now
  File "/usr/lib/python2.7/site-packages/tendrl/commons/utils/time_utils.py", line 2, in <module>
    from pytz import utc
ImportError: No module named pytz

However it looks like this package "pytz" usage is newly introduced into commons
in tendrl/commons/utils/time_utils.py file. I will remove this here and will add into commons spec.